index.js

```diff
@@ -7,25 +7,31 @@ const createDeferred = require('./deferred')
 
 const bundles = {}
 
-// by default, we transform JavaScript (up to anything at stage-4) and JSX
-const defaultOptions = {
-  webpackOptions: {
-    module: {
-      rules: [
+// by default, we transform JavaScript supported by @babel/preset-env
+const defaultBabelLoaderRules = () => {
+  return [
+    {
+      test: /\.js?$/,
```

Comment:
Did you mean to remove the `x`? This will match `.js` and `.j` files.

Reply:
Nope, this is a mistake that wasn't caught!
Apologies.